external/js/cothority/src/personhood/credentials-instance.ts

```diff
@@ -283,7 +281,10 @@ export class CredentialStruct extends Message<CredentialStruct> {
      * @param attribute     Name of the attribute
      * @param value         The value to set
      */
-    setAttribute(credential: string, attribute: string, value?: Buffer) {
+    setAttribute(credential: string, attribute: string, value?: Buffer | string) {
+        if (!(value instanceof Buffer)) {
+            value = Buffer.from(value ? value : "");
```

Comment:
```suggestion
            value = Buffer.from(value !== undefined ? value : "");
```

Or more readable, have a chain of
```
if (value === undefined)
else if (typeof value === "string")
else
```